src/components/datagrid/DataGrid.js

```diff
@@ -409,6 +409,17 @@ export default class DataGridComponent extends NestedArrayComponent {
     this.rebuild();
   }
 
+  focusOnNewRowElement(row) {
+    Object.keys(row).find((key) => {
+      const focusableElements = getFocusableElements(row[key].element);
```

Comment:
The inputs should be within a "refs.input" variable so this is not necessary and way less performant than it should be.